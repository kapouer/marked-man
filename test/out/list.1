A(1)                                                                      A(1)

NNAAMMEE
       aa

       This is a tight list:

              • ttooggggllee__ssttaattuuss: test

              • ssppaawwnn <<eexxeeccuuttaabbllee>> <<aaddddiittiioonnaall aarrggss>>
                TODO explain path-alike expansion,
                ability to get a block here.

       This is a loose list:

              • ttooggggllee__ssttaattuuss:
                test

              • ssppaawwnn <<eexxeeccuuttaabbllee>> <<aaddddiittiioonnaall aarrggss>>:
                TODO explain path-alike expansion,
                ability to get a block here.

                                 January 1979                             A(1)
