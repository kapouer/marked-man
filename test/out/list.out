[4mA[24m(1)                        General Commands Manual                       [4mA[24m(1)

[1mNAME[0m
       [1ma[0m

       This is a tight list:

        â€¢ [1mtoggle_status[22m: test

        â€¢ [1mspawn <executable> <additional args>[0m
          TODO explain path-alike expansion,
          ability to get a block here.

       This is a loose list:

        â€¢ [1mtoggle_status[22m:
          test

        â€¢ [1mspawn <executable> <additional args>[22m:
          TODO explain path-alike expansion,
          ability to get a block here.

                                 January 1979                             [4mA[24m(1)
